modules/commons/src/test/java/org/apache/synapse/commons/json/JsonValueTest.java

```diff
@@ -119,4 +120,13 @@ public void runFailTest(String value) {
             assert true;
         }
     }
+
+    public void testWriteAsJsonNullElement() {
+        try {
+            OutputStream out = Util.newOutputStream();
```

Comment:
Should we not close the stream at the end?

Reply:
Done